test/integration/targets/mysql_db/tasks/multi_db_create_delete.yml

```diff
@@ -0,0 +1,699 @@
+      - '{{ db1_name }}'
+      - '{{ db2_name }}'
+      - '{{ db3_name }}'
+    state: dump
+    target: '{{ dump1_file }}'
+    login_unix_socket: '{{ mysql_socket }}'
+  register: dump_result
+
+- name: assert successful completion of dump operation
+  assert:
+    that:
+       - dump_result.changed == true
+
+- name: run command to list databases like specified database name
+  command: mysql "-e show databases like 'database%';"
+  register: mysql_result
+
+- name: assert that databases does exist
+  assert:
+    that:
+      - "'{{ db1_name }}' in mysql_result.stdout"
+      - "'{{ db2_name }}' in mysql_result.stdout"
+      - "'{{ db3_name }}' in mysql_result.stdout"
+      - "'{{ db4_name }}' not in mysql_result.stdout"
+      - "'{{ db5_name }}' not in mysql_result.stdout"
+
+- name: state dump - file name should exist
+  file: name={{ dump1_file }} state=file
+
+
+# ==========================================================================
+# Dump all databases
+
+- name: state dump - dump2 file name should not exist
+  file: name={{ dump2_file }} state=absent
+
+- name: Dump existing databases
+  mysql_db:
+    name: all
+    state: dump
+    target: '{{ dump2_file }}'
+    login_unix_socket: '{{ mysql_socket }}'
+  register: dump_result
+
+- name: assert successful completion of dump operation
+  assert:
+    that:
+       - dump_result.changed == true
+
+- name: run command to list databases like specified database name
+  command: mysql "-e show databases like 'database%';"
+  register: mysql_result
+
+- name: assert that databases does exist
+  assert:
+    that:
+      - "'{{ db1_name }}' in mysql_result.stdout"
+      - "'{{ db2_name }}' in mysql_result.stdout"
+      - "'{{ db3_name }}' in mysql_result.stdout"
+      - "'{{ db4_name }}' not in mysql_result.stdout"
+      - "'{{ db5_name }}' not in mysql_result.stdout"
+
+- name: state dump - file name should exist
+  file: name={{ dump2_file }} state=file
+
+# ==========================================================================
+# Delete databases db1 and db3
+- name: Delete databases db1 and db3
+  mysql_db:
+    name:
+      - '{{ db1_name }}'
+      - '{{ db3_name }}'
+    state: absent
+    login_unix_socket: '{{ mysql_socket }}'
+  register: result
+
+- name: assert successful completion of deleting existing database
+  assert:
+    that:
+       - result.changed == true
+
+- name: run command to list databases like specified database name
+  command: mysql "-e show databases like 'database%';"
+  register: mysql_result
+
+- name: assert that specific databases which were deleted does not exist
+  assert:
+    that:
+      - "'{{ db1_name }}' not in mysql_result.stdout"
+      - "'{{ db2_name }}' in mysql_result.stdout"
+      - "'{{ db3_name }}' not in mysql_result.stdout"
+      - "'{{ db4_name }}' not in mysql_result.stdout"
+      - "'{{ db5_name }}' not in mysql_result.stdout"
+
+# ==============================IMPORT OPERATIONS ==========================
```

Comment:
```suggestion
# ==============================IMPORT OPERATIONS ==========================
```
I'd like to see it in another test file because it isn't related to the case

Reply:
Moved it to state_dump_import.yml file